machine-learning/03-medulloblastoma_PLIER.Rmd

```diff
@@ -210,22 +208,22 @@ plier_summary_df
 ```
 
 We can filter this `data.frame` based on significance of associations.
-We'll use `FDR < 0.05`.
+We'll use `FDR < 0.05`, where `FDR` is the False Discovery Rate. 
```

Comment:
Can we link to:

* [Wikipedia](https://en.wikipedia.org/wiki/False_discovery_rate)
* This [awesome StatQuest video](https://www.youtube.com/watch?v=K8LQSvtjcEo)

Here